include/runtime-support.h

```diff
@@ -29,37 +29,37 @@ namespace spicy::zeek::rt {
  * Exception thrown by event generation code if the value of an `$...`
  * expression isn't available.
  */
-class ValueUnavailable : public hilti::rt::UserException {
+class ValueUnavailable : public spicy::compat::UsageError {
 public:
-    using hilti::rt::UserException::UserException;
+    using spicy::compat::UsageError::UsageError;
```

Comment:
Does this actually work if `UsageError` is actually an `UserException`?

Reply:
I think it does, but maybe I'm missing something on why it shouldn't? Haven't seen any problems during testing yet. Will leave as is and see if something shows up.